src/EnergyPlus/DataEnvironment.hh

```diff
@@ -22,6 +22,7 @@ namespace DataEnvironment {
 	extern Real64 const EarthRadius; // Radius of the Earth (m)
 	extern Real64 const AtmosphericTempGradient; // Standard atmospheric air temperature gradient (K/m)
 	extern Real64 const SunIsUpValue; // if Cos Zenith Angle of the sun is >= this value, the sun is "up"
+	extern Real64 const StdBaroPressSeaLevel; // Standard barometric pressure at sea level (Pa)
```

Comment:
Sea level seems more clear to me (probably why I chose that for my constant name), but I'm fine with whatever as long as we all agree to use the same thing.